README.md

```diff
@@ -43,8 +43,8 @@ The application will start on [http://localhost:7080](http://localhost:7080). To
 The route is `/news/articles/:id`, where `id` is the filename of the static Article data, for example `c0000000001o`.
 
 Articles 25 & 27 are available on the Production environment as well as locally. These can be used for testing.
-[http://localhost:7080/news/articles/cl55zn0w0l4o](http://localhost:7080/news/articles/cl55zn0w0l4o)
-[http://localhost:7080/news/articles/cn7769kpk9mo](http://localhost:7080/news/articles/cn7769kpk9mo).
+[http://localhost:7080/news/articles/c6v11qzyv8po](http://localhost:7080/news/articles/c6v11qzyv8po)
+[http://localhost:7080/news/articles/c0g992jmmkko](http://localhost:7080/news/articles/c0g992jmmkko).
```

Comment:
Are these still "articles 25 and 27"? Is this meaningful to anyone? I wondered if it might be more helpful to explain their characteristics.

Would we not also want to link to the Persian one or is that premature?

Also does the link above to `c0000000001o` still have any value? It seems to be of limited use?

Reply:
I've made the changes to the text surrounding the examples. Hope that clarifies things somewhat.